src/ui/component/app/view.jsx

```diff
@@ -70,14 +70,14 @@ class App extends React.PureComponent<Props> {
     return (
       <div id="window" onContextMenu={e => openContextMenu(e)}>
         <Header />
-        <main className="page">
+        <section className="page">
```

Comment:
The `<Page>` component contained `<main>` too. It makes more sense to keep there.